core/src/main/scala/org/bitcoins/core/protocol/ln/LnPolicy.scala

```diff
@@ -0,0 +1,32 @@
+package org.bitcoins.core.protocol.ln
+
+import org.bitcoins.core.number.Int64
+import org.bitcoins.core.protocol.ln.currency.MilliSatoshis
+
+sealed abstract class LnPolicy {
+
+  /**
+   * The "amount_msat" field has been artificially limited to a UInt32. This means that the current maximum transaction that can be completed
+   * over the lightning network is 4294967295 MilliSatoshi.
+   * This is a self imposed limit, and is subject to change.
+   * Please see [[https://github.com/lightningnetwork/lightning-rfc/blob/master/02-peer-protocol.md#adding-an-htlc-update_add_htlc BOLT2]]
+    * for more info.
+   */
+  val maxAmountMSat: MilliSatoshis = MilliSatoshis(4294967295L)
+
+  val milliMultiplier: BigDecimal = BigDecimal(0.001)
+  val microMultiplier: BigDecimal = BigDecimal(0.000001)
+  val nanoMultiplier: BigDecimal = BigDecimal(0.000000001)
+  val picoMultiplier: BigDecimal = BigDecimal(0.000000000001)
```

Comment:
Perhaps this could be an ADT `sealed abstract class LnMultiplier { def multiplier: BigDecimal }`?